evm_loader/lib/src/account_storage.rs

```diff
@@ -86,6 +87,25 @@ impl<'rpc, T: Rpc + BuildConfigSimulator> EmulatorAccountStorage<'rpc, T> {
             Some(chains) => chains,
         };
 
+        // TODO: is it correct to get rent from the account
+        let mut rent_account = rpc
+            .get_account(&solana_sdk::sysvar::rent::id())
+            .await?
+            .value
+            .unwrap();
+        let rent = Rent::from_account_info(&AccountInfo {
```

Comment:
This can be simplified to
`let rent = bincode::deserialize(&rent_account.data)`

Reply:
Fixed